### ISSUE TITLE
Network configuration for listening port changed, broke systemd file

### ISSUE THREAD

**Issue by jmhickman** [Difegue/LANraragi#194]
**LRR Version and OS**
Native, Debian buster

**Bug Details**
At some point, LRR stopped accepting the network config (interface and port) on the command line, and needs it from an env var now? Dunno why that changed.

Just need to confirm that's the case and that I'm not doing something wrong when trying something like:  
`npm start http://*:80`

If so, I'll submit something to update the service file I wrote to support the env var.

**Comment by Difegue** [comment]
The launcher script used by `npm start` changed with the hypnotoad update, so I stopped accepting the network config as an argument since that was only really possible with the dumb morbo/daemon bridge that was in place beforehand.  

Please do use the LRR_NETWORK environment variable from now on.

**Comment by Difegue** [comment]
Closed by #199. Thanks!

**Comment by jmhickman** [comment]
Hmm, I noticed something that tickles my autism. Dunno how to solve it.

When starting the server with LRR_NETWORK=http://*:80, the log output at the beginning says:

`lanraragi npm[pid]: Server available at http://127.0.0.1:80`

While that is true...it isn't accurate. It should say something like "0.0.0.0" or the like, to indicate it's listening on all addresses. 

I tried searching through the project for that string to see where it's being built to have a go at modifying it, but all I got was results from the documentation.

Can this be addressed? It's good for admins to understand that their setting is in fact working as intended.
